register/templates/registration/login.html

```diff
@@ -0,0 +1,18 @@
+{% extends "registration/registration_base.html" %}
+{% load i18n %}
+
+{% block title %}{% trans "Confirm your email" %}{% endblock %}
```

Comment:
I'm not sure that step is a confirmation step, proposition: "Identify yourself", not very happy with it though, maybe something more on the vein of "Prove your identity" might be pertinent.